src/js/src/utils.js

```diff
@@ -17,27 +17,62 @@ function convertGeoJSONGeometryToOverPassGeometry({
     return `(poly: "${lineString}")`;
 }
 
-function createFormDataWithGeometry(shape, feature) {
+const createOverpassQueryElement = bbox => tag =>
+    `
+    node["${tag}"]${bbox};
+    way["${tag}"]${bbox};
+    relation["${tag}"]${bbox};
+    `;
+
+const createOverpassValueQueryElement = bbox => (tag, value) =>
+    `
+    node["${tag}"="${value}"]${bbox};
+    way["${tag}"="${value}"]${bbox};
+    relation["${tag}"="${value}"]${bbox};
+    `;
+
+const createOverpassDateQueryElement = (bbox, dateSelection) => tag =>
+    `
+    node["${tag}"]${bbox}(newer:"${dateSelection}");
+    way["${tag}"]${bbox}(newer:"${dateSelection}");
+    relation["${tag}"]${bbox}(newer:"${dateSelection}");
+    `;
+
+const createOverpassDateValueQueryElement = (bbox, dateSelection) => (tag, value) =>
```

Comment:
Is this common syntax for functions that return functions?  Took me a bit of scrolling up and down to put in all together wrt to declarations and usage.

Reply:
Yeah, it's the ES6 way to do currying or partial application. The lodash version of this would read a little more nicely but this project doesn't have that library and it didn't seem worthwhile to bring it in just for this.

I suppose these could be named more clearly to indicate that the initial functions are partially applied, but we can revisit if it becomes necessary.